### ISSUE TITLE
Hope to jump directly to the web page server

### ISSUE THREAD

**Issue by xp12047** [emad-elsaid/xlog#22]
Similar to this open source tool。[memo](https://github.com/mattn/memo)    
> When I enter `memo s`, it will directly open the browser without me having to copy the 127... address displayed on the cmd.  Help improve it.thanks

**Comment by emad-elsaid** [comment]
yeah looks like a good idea, I'm open for a PR that introduces that, would you like to contribute?

**Comment by xp12047** [comment]
> 我可能会在今天晚些时候处理它

thanks..
